docs/en/reference/configuration.rst

```diff
@@ -2,8 +2,7 @@ Installation and Configuration
 ==============================
 
 Doctrine can be installed with `Composer <http://www.getcomposer.org>`_.  For
-older versions we still have `PEAR packages
-<http://pear.doctrine-project.org>`_.
+older versions we still have `PEAR packages <http://pear.doctrine-project.org>`_.
```

Comment:
This subdomain doesn't exist anymore, is that expected? cc @jwage

Reply:
pear is no longer.

Reply:
I'll remove this link from the documentation.